### ISSUE TITLE
Twitter embeds stopped working

### ISSUE THREAD

**Issue by skoften** [spicywebau/craft-embedded-assets#156]
As of today, Twitter embeds stopped working; after copy pasting the link the review tries to load, giving this error;

![Screenshot 2020-12-18 at 11 37 11](https://user-images.githubusercontent.com/1286410/102614057-d5245e00-4133-11eb-8e51-5c1da6df6561.jpg)

![Screenshot 2020-12-18 at 13 18 22](https://user-images.githubusercontent.com/1286410/102614066-db1a3f00-4133-11eb-9f6d-94010a3baecf.jpg)

Tried on different servers, with different IP's.

Running Craft Pro 3.5.14
Embedded Assets 2.4.4

**Comment by ttempleton** [comment]
This is an issue with Embed v3, see oscarotero/Embed#421.

**Comment by ttempleton** [comment]
Some comments from the above-linked issue reported that setting a user agent in the `CurlDispatcher` gets Twitter embeds working again.  I've tested that with Embedded Assets and it has worked.  Since the fallback user agent Embed tries to use represents an old version of Chrome on macOS, here's how this can be worked around in Embedded Assets 2.4.4, `src/Service.php`, replacing the current lines 97-103, with the user agent for the current versions of Chrome/macOS:

```php
$curlDispatcherOptions = [
    CURLOPT_USERAGENT => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 11_1_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36',
];

if ($pluginSettings->referer) {
    $adapter = Embed::create($url, $options, new CurlDispatcher(array_merge($curlDispatcherOptions, [
        CURLOPT_REFERER => Craft::parseEnv($pluginSettings->referer),
    ])));
} else {
    $adapter = Embed::create($url, $options, new CurlDispatcher($curlDispatcherOptions));
}
```

I'll make a release by early next week one way or the other, whether there's an Embed v3 update we can enforce or we apply this workaround, but if you'd like to apply the workaround manually to get Twitter embeds working right now, feel free.  I'll also reopen this issue in the meantime.

**Comment by ttempleton** [comment]
Just released Embedded Assets 2.4.5, which sets the Embed v3 minimum requirement to 3.4.13, which fixes the Twitter error.
